heros/hero1.html.slim

```diff
@@ -0,0 +1 @@
+= partial('hero1.html.slim');
```

Comment:
OK, I see now. I think you need the partial for the initial HTML build of the page, and then you use this file in the swap function. Is that right?